dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/impl/service/identifiable/alias/UrlAliasServiceImpl.java

```diff
@@ -262,4 +266,33 @@ public UrlAlias update(UrlAlias urlAlias) throws CudamiServiceException {
       throw new CudamiServiceException("Cannot update urlAlias: " + e, e);
     }
   }
+
+  @Override
+  public void validate(LocalizedUrlAliases localizedUrlAliases) throws ValidationException {
+    if (localizedUrlAliases == null || localizedUrlAliases.isEmpty()) {
+      return;
+    }
+
+    Map<String, Integer> primaries = new HashMap<>();
+    localizedUrlAliases.flatten().stream()
+        .forEach(
+            u -> {
+              String key =
+                  u.getWebsite().getUuid() + "-" + u.getTargetUuid() + "-" + u.getTargetLanguage();
```

Comment:
```suggestion
                  (u.getWebsite() != null ? u.getWebsite().getUuid() : "default") + "-" + u.getTargetUuid() + "-" + u.getTargetLanguage();
```
Könnte `getWebsite` nicht auch `null` sein?